### PR TITLE
Remedial work

### PR DESCRIPTION
**Pull request by KristinaMatuleviciute**
Fixed linting errors
Updated dependencies
removed unneeded dependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,10 +29,18 @@
...
+    "lint": "lab test -dL",
+    "annotate": "docco github-auth.js -o doc"
+  },
   "homepage": "https://github.com/nherment/seneca-github-auth",
   "dependencies": {
-    "passport-github": "0.1.5",
-    "lodash": "~3.10.0"
+    "passport-github": "1.1.0",
+    "lodash": "~4.2.1",
+    "docco": "0.7.0",
```
Comment (ckiss) [inline package.json:47]: @KristinaMatuleviciute can you please move docco, eslint-\* and lab to devDependencies?

**Comment by ckiss** [comment]
looks good. just a small comment there and don't forget to add yourself as contributor @KristinaMatuleviciute

File: package.json (hunk 1 above)
Comment (ckiss) [inline package.json:47]: also you need all these three plugins installed: 
eslint-config-seneca
eslint-plugin-standard
eslint-plugin-hapi

after you install them, please run the linter again, you might find more linting issues

File: package.json [hunk 2]
```diff
@@ -22,17 +23,31 @@
     {
       "name": "Mircea Alexandru",
       "email":"mircea.alexandru@gmail.com",
-      "url" : "https://github.com/mirceaalexandru"
+      "url" : "https://github.com/mirceaalexandru",
```
Comment (ckiss) [inline package.json]: add a new object in the array for you, not in the same as Mircea's

File: doc/public/stylesheets/normalize.css [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+   ========================================================================== */
+
+/*
+ * Remove most spacing between table cells.
+ */
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
```
Comment (mihaidma) [inline doc/public/stylesheets/normalize.css]: missing new line at end of file

**Pull request merged**